modules/33acrossBidAdapter.js

```diff
@@ -194,18 +194,44 @@ function _createServerRequest({bidRequest, gdprConsent = {}, uspConsent, pageUrl
   // therefore in ad targetting process
   ttxRequest.id = bidRequest.bidId;
 
-  // Set GDPR related fields
-  ttxRequest.user = {
-    ext: {
-      consent: gdprConsent.consentString
-    }
-  };
-  ttxRequest.regs = {
-    ext: {
-      gdpr: (gdprConsent.gdprApplies === true) ? 1 : 0,
-      us_privacy: uspConsent || null
-    }
-  };
+  if (gdprConsent.consentString) {
+    ttxRequest.user = setExtension(
+      ttxRequest.user,
+      'consent',
+      gdprConsent.consentString
+    )
+  }
+
+  if (Array.isArray(bidRequest.userIdAsEids)) {
```

Comment:
is it worth to set the extension if the array is empty?

Reply:
It's not. I will add further check.